nbopenrefineproxy/handlers.py

```diff
@@ -33,6 +36,6 @@ def get(self, *args):
 def setup_handlers(web_app):
     web_app.add_handlers('.*', [
         (ujoin(web_app.settings['base_url'], 'openrefine/(.*)'),
-         OpenRefineProxyHandler, dict(state={})),
+         OpenRefineProxyHandler, dict(state={'port':3333})),
```

Comment:
What is the advantage of doing this?

Reply:
Thanks for merging!

3333 is the default port used by OpenRefine. Specifying it here makes it easier to connect with the python client (which tries to reach the OpenRefine server on 127.0.0.1:3333 by default). @psychemedia suggested this in https://github.com/betatim/openrefineder/issues/6#issuecomment-449406846

Reply:
Thanks. Makes sense!